src/clang-cpp-frontend/clang_cpp_language.cpp

```diff
@@ -21,8 +21,8 @@ languaget *new_clang_cpp_language()
 
 void clang_cpp_languaget::force_file_type()
 {
-  // We also force the standard to be c++98
-  compiler_args.push_back("-std=c++98");
+  // We also force the standard to be c++14
+  compiler_args.push_back("-std=c++14");
```

Comment:
This could be the default, but you should add a flag to set C++ standard as flag. Then you could duplicate the testcases with a C++98 passing and a C++14 KNOWNBUG

Reply:
I agree with @rafaelsamenezes' suggestion.

Reply:
Done.